scheduler/src/cook/mesos/data_locality.clj

```diff
@@ -198,8 +200,10 @@
           {:keys [job/uuid] :as job} (:job task-request)
           datasets (get-dataset-maps job)]
       (if-not (empty? (or datasets []))
-        (let [normalized-fitness (- 1.0
-                                    (get-in (get-data-local-costs) [datasets (.getHostname target-vm)] 1.0))
+        (let [{:keys [cost suitable]}(get-in (get-data-local-costs) [datasets (.getHostname target-vm)] {:cost 1.0
```

Comment:
Nitpick: missing space.

Reply:
Can we use `:or` in the destructuring to assign the default values for `cost` and `suitable`?

Reply:
Done